### PR TITLE
Encounter participant missing

### PR DESCRIPTION
**Pull request by anmolagar**
@sankarravi Please consider merging this pull request against issue no #153

### CODE REVIEW

File: lib/parsers/ccda/document.js [hunk 1]
```diff
@@ -61,6 +71,13 @@ Parsers.CCDA.document = function (ccda) {
       }
     },
     documentation_of: documentation_of_list,
+    encounterParticipant: {
+      name: performer_name_dict,
+      phone: {
+        work: performer_phone
+      },
+      address: performer_addr
+    },
```
Comment (sankarravi) [inline lib/parsers/ccda/document.js]: @anmolagar You're parsing data from `encompassingEncounter/encounterParticipant` but you're not actually using any of those parsed fields – this just outputs the data that's coming out of `documentationOf/performer` and is currently output in `documentation_of`.

It looks like very few of the samples are using the `encounterParticipant` field. Are you running into samples where this is important data?

Comment (anmolagar) [inline lib/parsers/ccda/document.js]: Yes, <code>encounterParticipant</code> is important data in our use case.

**Comment by anmolagar** [comment]
@sankarravi 
I have updated the pull request to rectify the issues you found earlier.
However, there is a failing test case in case of 'nist'. It seems like there are two different 'nist' xmls at different locations which are conflicting.
Both the nist files expects the output as <code>fixtures\json\nist_ccda_expected_output.json</code> while one has <code>encounterParticipant</code> and other do not.
I was not able to rectify this issue, please help me out here.

**Comment by anmolagar** [comment]
@sankarravi 
Please help me fix the test cases here. here is a failing test case in case of 'nist'. It seems like there are two different 'nist' xmls at different locations which are conflicting.
Both the nist files expects the output as fixtures\json\nist_ccda_expected_output.json while one has encounterParticipant and other do not.
I was not able to rectify this issue, please help me out here.

**Comment by sankarravi** [comment]
@anmolagar There is only one NIST file used for the tests. The test you are failing is the `NIST JSON -> CCDA -> JSON` test, which verifies that the parser and generator support the same data. To fix this test, you'll need to add support to `ccda_template.ejs` for outputting an `encounterParticipant`.

**Comment by anmolagar** [comment]
@sankarravi, even after adding code to ccda_template.ejs, i am getting test failure.
Looking for some help here as i am not an ejs expert.

**Comment by anmolagar** [comment]
@sankarravi 
Hi Ravi, any changes in <code>ccda_template.ejs</code> is not helping me to resolve the test case issue. Can you please have a look at this?

**Pull request closed without merging**